src/utilities/aspect-ratio/aspect-ratio.stories.mdx

```diff
@@ -92,9 +79,11 @@ Embeds may be combined with [rounded corner utilities](/docs/utilities-rounded-c
   <Story
     name="Rounded"
```

Comment:
I'm not sure this example makes sense anymore since it's not really relevant to this type of utility. We can probably remove it.